src/main/java/com/google/devtools/build/lib/sandbox/LinuxSandboxedSpawnRunner.java

```diff
@@ -178,6 +184,12 @@ protected SandboxedSpawn prepareSpawn(Spawn spawn, SpawnExecutionContext context
     sandboxExecRoot.getParentDirectory().createDirectory();
     sandboxExecRoot.createDirectory();
 
+    Path sandboxTmp = null;
+    if (getSandboxOptions().sandboxHermeticTmp && !getSandboxOptions().sandboxTmpfsPath.contains(PathFragment.create("/tmp"))) {
```

Comment:
Should this check for starting with `/tmp` (or rather either being `/tmp` or starting with `/tmp/`) instead of containing `/tmp`? Not sure what would happen if `--sandbox_tmpfs_path` is `/tmp/foobar`.

Reply:
Added a check for this as well as a descriptive warning. I think we should be able to support it by sorting the tmpfs paths into the bind mounts, but would prefer to ignore this edge case for now.